wfe/wfe.go

```diff
@@ -232,6 +233,17 @@ func (wfe *WebFrontEndImpl) sendError(prob *acme.ProblemDetails, response http.R
 	response.Write(problemDoc)
 }
 
+func (wfe *WebFrontEndImpl) RootCert(
+	ctx context.Context,
+	logEvent *requestEvent,
+	response http.ResponseWriter,
+	request *http.Request) {
+
+	response.Header().Set("Content-Type", "application/pem-certificate-chain; charset=utf-8")
+	response.WriteHeader(http.StatusOK)
+	_, _ = response.Write(wfe.ca.GetRootCert().PEM())
```

Comment:
Can you add a check that `wfe.ca.GetRootCert() != nil` and return an error if it is? I don't think there is a way to get Pebble running and have that field be nil but it would be nice to avoid the nil panic on `wfe.ca.GetRootCert().PEM()` just in case.